models/semantic_segmentation/DUC/README.md

```diff
@@ -0,0 +1,72 @@
+
+### Output
+The output of the network is a tensor of shape (1 X `label_num` X `H`*`W`) where `H` and `W` are the height and width of the output segmented map.
+
+### Postprocessing
+The output tensor is reshaped and resized to give the softmax map of shape (`H` X `W` X `label_num`). The raw label map is computed by doing an argmax on the softmax map which is used to generate the colorized segmented images. Check [duc-postprocess.py](duc-postprocess.py) for code.
+
+<!--
+To do quick inference with the model, check out [Model Server](https://github.com/awslabs/mxnet-model-server/blob/master/docs/model_zoo.md/#arcface-resnet100_onnx).
+-->
+
+## <a name="dset"></a>Dataset
+Cityscapes dataset is used for training and validation. It is a large dataset that focuses on semantic understanding of urban street scenes. It contains 5000 images with fine annotations across 50 cities, different seasons, varying scene layout and background. There are a total of 30 categories in the dataset of which 19 are included for training and evaluation. The training, validation and test set contains 2975, 500 and 1525 fine images, respectively.
+
+### Download
+First, go to the [Cityscapes download page](https://www.cityscapes-dataset.com/downloads/) and register for an account (login if account already made). Next, find and download the following two files:
+
+|Filename                 | Size  | Details|
+|-------------------------|:------|:-------|
+|leftImg8bit_trainvaltest.zip| 11 GB| train/val/test images|
+|gtFine_trainvaltest.zip  | 241 MB| fine annotations for train and val sets|
+
+### Setup
+* Unpack the zip files into folders `leftImg8bit_trainvaltest` and `gtFine_trainvaltest`.
+* Use the path to the train/val folders inside these folders for training/validation.
+* Use the files [val.lst](https://s3.amazonaws.com/onnx-model-zoo/duc/val.lst) and [train.lst](https://s3.amazonaws.com/onnx-model-zoo/duc/train.lst) containing image lists required during validation and training respectively.
+
+Please note that the dataset is under copyright. Refer to the [citation](https://www.cityscapes-dataset.com/citation/) page for details.
+
+## Validation accuracy
+The [mIOU](#metric) score obtained by the models on the validation set are mentioned above and they match with those mentioned in the paper.
+
+## Training
+Coming soon.
+
+## <a name="metric"></a>Validation
+The metric used for validation is **mean Intersection Over Union (mIOU)**. For each class the intersection over union (IOU) of pixel labels between the output and the target segmentation maps is computed and then averaged over all classes to give us the mean intersection over union (mIOU).
```

Comment:
You should define this earlier. I read mIOU in the overview and didn't know what it was.

Reply:
By earlier do you mean earlier in the readme for DUC? In order to stick to the readme template I defined the metric below and linked all previous instances of mIOU to that section.. does that work?